pandas/io/pytables.py

```diff
@@ -1,27 +1,37 @@
+# pylint: disable-msg=E1101,W0613,W0603
 """
 High level interface to PyTables for reading and writing pandas data structures
 to disk
 """
 
-# pylint: disable-msg=E1101,W0613,W0603
-from datetime import datetime, date
-import time
-import re
 import copy
 import itertools
-import warnings
 import os
+import re
+import time
+import warnings
+
+from datetime import datetime, date
 from distutils.version import LooseVersion
 
 import numpy as np
 
+from pandas import (Series, DataFrame, Panel, Index,
+                    MultiIndex, Int64Index, isna, concat, to_datetime,
+                    SparseSeries, SparseDataFrame, PeriodIndex,
+                    DatetimeIndex, TimedeltaIndex)
+from pandas import compat
 from pandas._libs import algos, lib, writers as libwriters
 from pandas._libs.tslibs import timezones
-
-from pandas.errors import PerformanceWarning
-from pandas import compat
 from pandas.compat import u_safe as u, PY3, range, lrange, string_types, filter
```

Comment:
`u` is no longer needed, can be replaced by `u` literal

Reply:
Replaced all occurrences with the literal for this file